.changes/next-release/bugfix-sso-68327.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "bugfix",
+  "category": "``sso``",
+  "description": "Support the ``--ca-bundle`` option on SSO commands."
```

Comment:
Can we also add that we support the ``--no-verify-ssl`` option now?

Reply:
Added via https://github.com/aws/aws-cli/pull/9112/commits/7ea5e364410bc8e4c1ed63d7e87bf7f566b66816 (changelog message too).